python/ray/air/integrations/mlflow.py

```diff
@@ -318,7 +318,7 @@ def log_trial_end(self, trial: "Trial", failed: bool = False):
 
         # Log the artifact if set_artifact is set to True.
         if self.should_save_artifact:
-            self.mlflow_util.save_artifacts(run_id=run_id, dir=trial.local_path)
```

Comment:
`local_path` is expected here as this requires saving local artifacts. The proposed `path` often points to non-local (e.g. cloud) paths.

Reply:
Hi @matthewdeng , thank you for taking a look.

It's been awhile, but in my testing for #46569 checkpoints were not saved to this temporary directory pointed to by `local_path` and thus do not get pushed up to the server.

To fix #46569 should changes be made to where checkpoints are saved then?